### DISCUSSION TITLE
Creating a interface for Translator's convenience

### DISCUSSION THREAD

**Discussion by dvrsaran** [organicmaps/organicmaps#3916] [Ideas]
I have an idea about inserting a new translated word into the .txt "String.txt". currently, we are typing each word into the raw text file. In that case, we create the interface for the below-mentioned operations,
1) Read the .txt file and choose by the user(translator) which word you want to translate or randomly given word.
2) Choose the language (which language you want to translate).
3) Giving suggestions from google translate, check if is that correct or not(from the translator side). 
In this way, we can speed up the process.

**Comment by biodranik** [comment]
There are already many services like this, the issue is only with proper integration with them, so it won't take too much of our time. And we don't need to speed up the process. We need _quality_ translations.

**Reply by dvrsaran** [reply]
Here I mean speed up work performance without any compromise in quality. There are lots of languages ​​(growing database) every time the translator scrolls through, finds his language (arranged alphabetically), and puts a keyword (en= | ta=) before his translation, In this process, we need to optimize that.

**Reply by rtsisyk** [reply]
> Here I mean speed up work performance without any compromise in quality. There are lots of languages ​​(growing database) every time the translator scrolls through, finds his language (arranged alphabetically), and puts a keyword (en= | ta=) before his translation, In this process, we need to optimize that.

What do you propose? Could you please show some example of projects where this work is organized in a better way? Probably we can learn something new and adopt.
